helm/daq-config-server/templates/deployment.yaml

```
@@ -53,14 +53,19 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
-          {{- with .Values.volumeMounts }}
           volumeMounts:
-            {{- toYaml . | nindent 12 }}
-          {{- end }}
+          - name: config-volume
+            mountPath: "/etc/config"
+            subPath: config.yaml
```

```suggestion
          - name: config-volume
            mountPath: "/etc/config"
            subPath: config.yaml
          {{- with .Values.volumeMounts }}
            {{- toYaml . | nindent 12 }}
          {{- end }}
```

Leaves the user open to configuring more mounts e.g. see conversation on blueapi